### ISSUE TITLE
Basic mobs don't seem hostile any more, simple_mobs unaffected

### ISSUE THREAD

**Issue by iprice** [tgstation/tgstation#84722]
Reporting client version: 515.1640

## Round ID:
231719

## Testmerges:
- [[TM ONLY] Hijacked Revision Date for v3 TGS Api](https://www.github.com/tgstation/tgstation/pull/83227)
- [Limits the methods through which you can be stunned by a shove. Shoves disarm people who are knocked down.](https://www.github.com/tgstation/tgstation/pull/84640)

## Reproduction:

spawn a /mob/living/basic/trooper/syndicate/ranged/shotgun/space
spawn a /mob/living/basic/trooper/pirate/melee/space
spawn a /mob/living/basic/bear/russian

Enjoy your new friends, they wont attack you even if you attack them

spawn a /mob/living/simple_animal/hostile/mimic/crate
get attacked some

This has notable impact on space ruins etc

**Comment by SyncIt21** [comment]
This has been happening actually for years actually.

I remember spawning gorilla's in clown planet sometime back in 2022 to kill everyone there and they didn't do anything when attacked.

The only explanation is that AI subsystem doesn't tick when there is lag elsewhere in other subsystems.

**Comment by necromanceranne** [comment]
Yeah this is a priority issue. I believe a few prs were tested to change the priorities amd it broke things pretty bad.

**Comment by iprice** [comment]
One really odd thing was they would /move/, not in a targetted way, just wandering, but wouldn't react to attacks.

It's like whatever implements the current AI behaviour ticks and makes it continue to wander, but whatever makes the AI think and update its state doesn't happen (and yet I'd expect being attacked to create change in the AI state there and then rather than on some later tick, which would then manifest in the still happening action ticks)

**Comment by san7890** [comment]
we just have too many AIs to process now. we tried culling it with turning them idle if no players are on a given z-level but the majority of basic mob refactors have really chewed up that portion of the tick
